client/src/auth/requiresAuth.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import axios from "axios";
+
+axios.defaults.baseURL = "http://localhost:9000/api";
```

Comment:
You could export an axios object with this so that you don't have to repeat yourself elsewhere:
```javascript
import axios from "axios";

axios.defaults.baseURL = "http://localhost:9000/api";

module.exports = axios;
```
or, using an [axios instance](https://github.com/axios/axios#creating-an-instance):
```javascript
import axios from "axios";

const axiosInstance = axios.create({
  baseURL: "http://localhost:9000/api"
});

module.exports = axiosInstance;
```